phpstan.neon

```diff
@@ -0,0 +1,8 @@
+parameters:
+    level: 7
+    bootstrapFiles:
+       - tests/bootstrap.php
+    checkGenericClassInNonGenericObjectType: false
+    checkMissingIterableValueType: false
+    ignoreErrors:
+            - '#Call to .+ on an unknown class Generated.+AutoCompletion#'
```

Comment:
This can get fixed with proper test harness as per our docs!

Reply:
https://spryker.atlassian.net/wiki/spaces/CORE/pages/2868248862/Standalone+module+testing
and as per most other standalone modules that have a valid test harness set up

Reply:
Based on this doc there some example modules with valid settings https://spryker.atlassian.net/wiki/spaces/CORE/pages/2868248862/Standalone+module+testing#Example-integrations

but all of them have `Call to .+ on an unknown class Generated.+AutoCompletion` in `phpstan.neon`

Reply:
Those must be the outdated onces
There are at least several that showcase the valid setup
https://github.com/spryker-eco/loggly

Reply:
Thanks!
But `Loggly` does not have `DependencyProvider`, so that's the reason it does not have such ignored error. 
We get the error for DP, not for tests

Reply:
Hm, I am 100% sure there is one, because I personally fixed it at the time
Running the IDE generation itself would solve this afaik.